blocks/share-bar-block/_index.scss

```diff
@@ -0,0 +1,32 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-share-bar {
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+	@include scss.block-properties("share-bar");
+	@include scss.block-components("share-bar");
+}
+
+.b-share-bar__facebook {
+	@include scss.block-properties("share-bar-facebook");
+	@include scss.block-components("share-bar-facebook");
+}
+
+.b-share-bar__linkedin {
+	@include scss.block-properties("share-bar-linkedin");
+	@include scss.block-components("share-bar-linkedin");
+}
+
+.b-share-bar__pinterest {
+	@include scss.block-properties("share-bar-pinterest");
+	@include scss.block-components("share-bar-pinterest");
+}
+
+.b-share-bar__twitter {
+	@include scss.block-properties("share-bar-twitter");
+	@include scss.block-components("share-bar-twitter");
+}
+
+.b-share-bar__email {
```

Comment:
There is a miss match with the code and the tokens - the code has the following class `b-share-bar__envelope`